jme3-core/src/main/java/com/jme3/anim/AnimComposer.java

```diff
@@ -314,7 +314,7 @@ public Action removeAction(String name) {
      * @param mask the desired mask for the new layer (alias created)
      */
     public void makeLayer(String name, AnimationMask mask) {
-        AnimLayer l = new AnimLayer(this, name, mask);
+        AnimLayer l = new AnimLayer(name, mask);
```

Comment:
Please create a separate PR for optimizations.

Reply:
To me, this was not a separate optimization but removing an essentially unnecessary field that would make serialization more difficult.

Reply:
ok, if the change is essential for new functionality, it's good to go

Reply:
It's correct. It's redundant, but it also makes serialization difficult (impossible due to SO?)